vendor/engines/saml_authentication/config/locales/en.yml

```diff
@@ -0,0 +1,6 @@
+en:
+  views:
+    saml_authentication:
+      sessions:
+        new:
+          sso_button: Single Sign On
```

Comment:
I think in the RaDar fork it might be friendlier to say something like "Log in with NetID" or whatever the Dartmouth equivalent is.

Reply:
Yeah, that'll probably make sense. Once we have this up on stage, I'll ask them if they have an example of an app that uses this kind of authentication so we can see what other apps say.